jobspec/parse_service.go

```diff
@@ -0,0 +1,507 @@
+package jobspec
+
+import (
+	"fmt"
+
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/hcl"
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/helper"
+	"github.com/mitchellh/mapstructure"
+)
+
+//TODO(schmichael) combine with non-group services
+func parseGroupServices(g *api.TaskGroup, serviceObjs *ast.ObjectList) error {
```

Comment:
Phooey, I think I goofed in #6045. I thought I had made `parseServices` usable for both group and task services so this could be removed.

I'll try to fix it in a followup unless you want to sneak it into this PR.